create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/conversion/ContributorExtractor.java

```diff
@@ -83,13 +80,13 @@ private void extractContributorFromAuthorOrCollaboration(Object authorOrCollabor
     }
 
     private void replaceExistingContributor(Contributor existingContributor, AuthorGroupTp authorGroupTp) {
-        //TODO: when contributor with cristin id's have been enriched with cristin affiliations, then this step
-        // should be skipped
-        var optionalNewAffiliations = generateAffiliation(authorGroupTp);
-        optionalNewAffiliations.ifPresent(
-            organizations ->
-                createNewContributorWithAdditionalAffiliationsAndSwapItInContributorList(organizations,
-                                                                                         existingContributor));
+        if (Objects.isNull(existingContributor.getIdentity().getId())) {
```

Comment:
Can existingContributor or its getIdentity also be null?

Reply:
existingContributor and getIdentity can not be null, however getId() might be null.